Anime-App/Feature/Anime/View/AnimeViewController.swift

```diff
@@ -9,8 +9,14 @@ import UIKit
 
 class AnimeViewController: UIViewController {
     
+    
+    
     var searchTask: DispatchWorkItem?
     private var animeViewModel = AnimeViewModel()
+    private var page = 10
+    private var pageLimit = 20
+    private var isLoading = false
```

Comment:
Do you even need this isLoading? What is it even doing? It doesn't look like you use it for anything. Again Tobi this is messy and lazy. You've really got to cut this out. This is the type of stuff that will frustrate people when they review your work.